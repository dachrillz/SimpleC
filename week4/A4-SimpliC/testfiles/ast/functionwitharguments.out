Program
  List
    Function
      IdDecl ID="a"
      List
        IdDecl ID="b"
        IdDecl ID="c"
        IdDecl ID="d"
      Block
        List
          Return
            Numeral NUMERAL="0"
