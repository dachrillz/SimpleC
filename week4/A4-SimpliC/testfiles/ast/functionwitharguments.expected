Program
  List
    Function
      IntType
      IdDecl ID="a"
      List
        IdDecl ID="b"
        IdDecl ID="c"
        IdDecl ID="d"
      Block
        List
          Return
            Numeral NUMERAL="0"
